plugin/pkg/global/deletionconfirmation/admission_test.go

```diff
@@ -384,11 +380,11 @@ var _ = Describe("deleteconfirmation", func() {
 				})
 
 				It("should succeed for true annotation value (live lookup)", func() {
-					attrs = admission.NewAttributesRecord(nil, nil, core.Kind("ShootState").WithVersion("v1alpha1"), shootState.Namespace, shootState.Name, v1alpha1.Resource("shootstates").WithVersion("v1alpha1"), "", admission.Delete, &metav1.DeleteOptions{}, false, nil)
+					attrs = admission.NewAttributesRecord(nil, nil, core.Kind("ShootState").WithVersion("version"), shootState.Namespace, shootState.Name, gardencorev1beta1.Resource("shootstates").WithVersion("version"), "", admission.Delete, &metav1.DeleteOptions{}, false, nil)
 
 					Expect(shootStateStore.Add(&shootState)).NotTo(HaveOccurred())
 					extGardenClient.AddReactor("get", "shootstates", func(action testing.Action) (bool, runtime.Object, error) {
-						return true, &v1alpha1.ShootState{
+						return true, &gardencorev1beta1.ShootState{
```

Comment:
This should use the internal version, right?

Reply:
oh I missed that.